### PR TITLE
Closes AgileVentures/MetPlus_tracker#575

### PR DESCRIPTION
**Pull request by jeanwei**
Js pagination using 'bootstrap-will_paginate' gem is implemented

### CODE REVIEW

File: app/helpers/remote_link_pagination_helper.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module RemoteLinkPaginationHelper
+  class BootstrapLinkRenderer < WillPaginate::ActionView::BootstrapLinkRenderer
+    def link(text, target, attributes = {})
+      attributes['data-remote'] = true
+      super
+    end
+  end
+end
```
Comment (patmbolger) [inline app/helpers/remote_link_pagination_helper.rb]: Is this being used anywhere?

Comment (jeanwei) [inline app/helpers/remote_link_pagination_helper.rb]: this helper is only used in one place: app/helpers/application_helper.rb

Comment (patmbolger) [inline app/helpers/remote_link_pagination_helper.rb]: Of course!  How did I miss that?  Nicely structured!

**Comment by patmbolger** [comment]
Looks good - merging

**Pull request merged**